### ISSUE TITLE
Proposal for a JSConf NZ

### ISSUE THREAD

**Issue by jenofdoom** [JavaScript-NZ/Society-Meetings#35]
I've talked a bit about how I'd like to run a JS community conference, under the umbrella of JavaScript NZ. This issue is my attempt at capturing what I'm trying to suggest, so that we can discuss it some more.
## What do I mean by a community conference?
- Low cost ($100ish full price tickets, $50ish student/unwaged tickets)
- Local (mostly NZ based) speakers
- Encourage first time speakers
- 2 days, variety of types of interaction - talks, sprints, tutorials
- Variety of difficulty levels from beginner stuff through to expert
- Aim for 200 to 500 ish participants
- Because of low ticket cost, sponsorship would be necessary
- Rotating location
## What do I think we'd need to do to be able to do this?

If people are keen, there are obviously some things we'd need to make some calls about. Here's my thoughts about some of the things that would need organising:
### City

I've suggested above that we'd rotate between cities, because I think that's a great way of getting a diverse range of attendees who might not normally be able to attend due to travel costs. We'd need to pick a location for the first conf, though, with Auckland, Wellington, Christchurch or Dunedin probably being the obvious contenders.
### JSConf + NodeConf = one conference?

There has been some discussion on the slack channel about the possibility of another NodeConf. I don't really have a clear feeling about whether we'd one to do one conf annually that would be big-umbrella JS (with maybe a specific track or day dedicated to Node?) or if it's better to keep it a separate event. In my mind having one event has the advantage that we don't end up cannibalising the sponsorship from the other event, plus it might be a good gateway into Node stuff for those who wouldn't otherwise attend that... but there is a definite focus change so that could be considered undesirable. I would love to hear the thoughts of folks interested in organising NodeConfs one way or the other.
### Conference director and conference committee

To run an event like this requires both a heap of time and a group of committed people. I'd be happy to be the conference director if we run this in Wellington if no-one else had their heart set on it (I've got relevant experience in that I directed Kiwi PyCon 2014), but if we picked a different city the director should be local.
### Sponsorship

This is key. We would need in the region of 30 to 50k in sponsorship income (depending on if we go with a catered model, it's one of the biggest line items). I think this is totally achievable, especially given how many companies are using JS nationally. A tiered gold/silver/bronze system works quite nicely.
### Time of year

We'd need at least a year to get something sorted. So Nov/Dec 2016 is probably the earliest we could aim for. I actually think Nov is quite light on tech events so could be a great month, but open to suggestions here.
## Other conferences we can take inspiration from

There is obviously all at other JSConf events globally, which you can see more about at [http://jsconf.com/](http://jsconf.com/). If we wanted to use the official JSConf event name, we'd need to follow [their rules](http://jsconf.com/i-want-to-run-a-jsconf.html).

I'm taking a lot of inspiration from the [Kiwi PyCon model](https://nzpug.org/kiwipycon/what-is-kiwipycon/) - that's how I know it is possible to organise one of these conferences cheaply with a focus on community involvement.

[One Shot](http://oneshot.nodeconf.com/) is the template for the most recent Christchurch Node conference.

**Comment by jenofdoom** [comment]
Keen to hear other people's thoughts, this was just what was in my brain, I expect we'll have to iterate on it a tonne :)

**Comment by phenomnomnominal** [comment]
Looking at those JSConf rules, number 1 is "You must have attended an existing JSConf event." Do we have anyone who has done so? Seems like a potentially expensive barrier to entry.

**Comment by Unfocused** [comment]
I attended JSConf.au 2014, if that counts (and know some of the organisers, if that helps).

**Comment by Unfocused** [comment]
And in terms of Jen's thoughts, this is exactly what I'd love to see.

**Comment by anotheredward** [comment]
Very interested in the idea of NodeConf === JSConf.
~6/10 talks from NodeConf were either language agnostic, or Javascript, so lots of healthy overlap.

**Comment by ndorfin** [comment]
@jenofdoom By encouraging first-time speakers, wouldn't that result in poor quality talks? Or are you thinking of screening and training the first-time speakers?

**Comment by digitalsadhu** [comment]
When you consider budget, I would highly advise breaking costs into fixed and per person costs. Your fixed costs are things like your venue which is basically the same no matter how many people come (give or take) Your per person costs being food, swag etc etc.
Once you know your fixed costs, thats what you aim for with sponsorship $$. Ticket price should only go to per person costs and allows you to scale the ticket price depending on how much you want to provide each person.
This also provides a go/no go point for the conference. If you can't get the sponsorship for the fixed costs, the conference doesn't happen. This way you are totally safe from ending up with a bill at the end that you have to pay yourselves.

**Comment by PrototypeAlex** [comment]
Nice work on getting this down in an issue @jenofdoom 

I like the idea of heavy sponsorship to reduce the ticket cost, @buildmaster might have a better idea on how viable the $50-100 tickets would be.

I'm for the majority of locally grown speakers, do you think you'd run it as an RFP type thing or shoulder tap people? 

I haven't attended a JSConf, but have listened to this JSJabber and totally jumped on board with what they're trying to do - https://devchat.tv/js-jabber/105-jsj-jsconf-and-organizing-conferences-with-chris-williams (It's a really good listen if you haven't attended before)

Lastly, November would be an interesting time of year, it's normally Feb-April where all the conferences are crammed into, so having one at the end of the year would be refreshing.

In short I love the idea of encompassing an all in one conference to double up on resources.

:+1:

**Comment by buildmaster** [comment]
Hey All,

WDCNZ is a very different style of conference. We put a lot of money into 'production' values such as AV, getting people to NZ, showing off NZ as a place to those speakers (we fund good accomodation and travel)) we also don't use the cheapest venue around (Positively Wellington Venues are great but they aren't the cheapest) So I'm not sure how much help I can give. Our costs get close to $200k once it's all added up so you can certainly do it cheaper.

We're also thinking of moving WDCNZ to September/October time in 2016 as we have some logistical issues making it harder to do one earlier in the year.

I've been toying with the idea of perhaps making a side JSConf style event to round out the week after WDCNZ but haven't taken it any further than just "that might be nice". I know there are requirements around calling an event a JSConf event that I wouldn't meet myself (mainly I've never attended a JSConf event) but would happily try and work something with someone else.

**Comment by jenofdoom** [comment]
@phenomnomnominal and @Unfocused, yes the "you've been to one already" is something we'd have to work through, it's great that @Unfocused has been to the AU one already and knows folks that helped organise that, we can probably get a lot of pointers from that. Whoever ends up on the committee would probably have to go over the ditch to visit the 2016 AU one.

@ndorfin  re: encouraging first time speaker, yep, you do sometimes get a bit of a quality hit. In a multi-track conference that's often not as bit a deal as you'd think, though. We do have to provide resources ("here is how you write a proposal" etc) to help, and encouraging them to practise at smaller local meetups like e.g. React meetup helps a lot (funnily enough I think the bar for the main JS meetup is already pretty high, for Welly at least, potentially higher that it would be for this conf).

@digitalsadhu  yep, I totally agree about splitting fixed vs per participant costs, and I managed the budget for the previous conference that I ran in exactly such a way. As you say, it helps to balance the budget because essentially your sponsorship just ends up taking care of the fixed costs like venue, and your confence can scale depending on the number of tickets you sell more gracefully.

@PrototypeAlex  I'm definitely imagining Request For Proposals, it's a great way of reaching out to people you'd never think of otherwise.

@buildmaster yep, WDCNZ is pretty different from this (although for the record it's tied with Kiwicon as my favourite conference to attend personally). Thanks for the heads up about moving it to September/October though, I think we'd definitely want to take that into consideration when planning the dates for this potential conf.

There definitely ARE much cheaper venues that Positively Wellington venues, and actually in some cities other than Wellington some of the Universities are amenable to providing venue sponsorship i.e. no cost for venue (I never had any luck with that in Welly, but it's always worth another try).

It is also possible to do do recording at a slightly reduced quality compared to WDCNZ or Webstock but much much cheaper using an on the fly video editing workflow with volunteer labour. [NextDay](http://nextdayvideo.com/) are the company I've used for this before and they are fantastic. As the name suggests the videos go up in a pretty timely fashion too :)

**Comment by jenofdoom** [comment]
Hi folks, so at the last committee meeting we realised we should form a subcommittee to take this forwards. I'm hoping that some of you will want to be involved going forwards. I've set up a Loomio group at https://www.loomio.org/g/quzZfV7C/javascript-conference-nz , if you request an invite there I'll grant anyone who would like it access.

**Comment by PapaBearNZ** [comment]
Sequeing off @jenofdoom , If Hamilton were a consideration for the location, I know that Wintec are often happy to provide a venue and support with av, etc.  They were a great help with the Innes48 startup weekend.

**Comment by jenofdoom** [comment]
@PapaBearNZ we've got a thread on potential cities right now on the Loomio board if you'd like to chip in :)

Right now the suggestion is Wellington, but if we'd like it to be another city instead all we'd need is someone (or someones) to step forward from that city to drive the development.

Once we've run one event this year, we should hopefully end up with a "bible" or resources to make it easy to shift the event to another city for 2017.

**Comment by jenofdoom** [comment]
Closing this issue in favour of some more specific tickets. Once again, if you are interested in helpng out, join the Loomio group at https://www.loomio.org/g/quzZfV7C/javascript-conference-nz or join the #js-conf channel on Slack.
